app/controllers/catalog_controller.rb

```diff
@@ -31,14 +31,33 @@ class CatalogController < ApplicationController
     config.document_solr_path = 'get'
     config.document_unique_id_param = 'ids'
 
-    # solr field configuration for search results/index views
-    config.index.title_field = 'cho_title_ssim'
+    locale_encoded_fields = lambda do |field_prefix, suffix = 'ssim'|
+      (Settings.acceptable_bcp47_codes << 'none').map do |code|
+        [code, "#{field_prefix}.#{code}_#{suffix}"]
+      end.to_h
+    end
+
+    negative_range_query_for_language_fields = lambda do |field_prefix, suffix = 'ssim'|
```

Comment:
I kept this in a separate commit so it can easily be popped off when #641 is merged (but I had already written the code when @cbeer did that, so why not)

Reply:
Removed this commit so we don't have this query going on any longer (and the Empty values facet will be correct after content is reindexed).